pkg/registry/apis/datasource/register.go

```diff
@@ -6,6 +6,7 @@ import (
 	"fmt"
 
 	"github.com/grafana/grafana-plugin-sdk-go/backend"
+	"github.com/grafana/grafana/pkg/infra/log"
```

Comment:
I see this elsewhere in registry apis, but wonder if we should be using `"github.com/grafana/grafana-plugin-sdk-go/backend/log` so there is one less dependency on grafana/grafana

Reply:
hmmm... I like the idea of removing our dependencies here on grafana/grafana but I don't know that using the plugin sdk's logger is the right call for that since this isn't a plugin and it looks like that logger doesn't format properly outside of the context of a plugin: 
![Screenshot 2024-08-29 at 8 54 58 AM](https://github.com/user-attachments/assets/784e5674-5b7c-4c68-9979-065aa34fb9d6)